app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,24 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource only: [:create]
+  layout false
+
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
We should extract it into concern and use in `ajax` specific controllers (right now in `CommentsController` and `DiscussionsController`)